### ISSUE TITLE
JS Stubs depend on javaType.

### ISSUE THREAD

**Issue by mcarcaso** [foam-framework/foam2#1516]
This line here makes stripping the `javaType` property from `foam.core.Argument`s impossible:
https://github.com/foam-framework/foam2/blob/master/src/lib/Stub.js#L21

**Comment by adamvy** [comment]
I don't think we every want to serialize contexts, so if we have a foam.typeOf() for Context then we should just be able to always null them out as arguments.

I also think we should add contextOriented: true to Method so we can better detect this (it would also clean up some hacks in the java Skeleton code).
